HelpfulItem.h

```diff
@@ -7,9 +7,9 @@
 #include "DragonSlayer.h"
 #include "Dragon.h"
```

Comment:
you're not using these in this header file.  don't include what you don't use
- [ ] revise